src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -40,6 +45,16 @@ private RemoteQuerySessionManager(RemoteQueryDatum queryDatum,
                                       EvaluationContext evaluationContext) {
         this.queryDatum = queryDatum;
         this.evaluationContext = evaluationContext;
+        initUserAnswers();
+    }
+
+    private void initUserAnswers() {
+        OrderedHashtable<String, QueryPrompt> queryPrompts = queryDatum.getUserQueryPrompts();
+        for (Enumeration en = queryPrompts.keys(); en.hasMoreElements(); ) {
+            String promptId = (String)en.nextElement();
```

Comment:
I think this needs to get updated to make sure this pr is included:
https://github.com/dimagi/commcare-core/pull/959

which has the init behavior for the query screen that USH needs for performing de-dupe.

Reply:
Thanks for flagging, my master was a couple of commits behind HEAD. That PR is now included, along with a few more commits from @shubham1g5 and @stephherbers

Reply:
great, thanks. Reviewed the new commits and didn't see any UserDB changes in those either.